atox/src/main/kotlin/ui/contactprofile/ContactProfileViewModel.kt

```diff
@@ -1,4 +1,4 @@
-// SPDX-FileCopyrightText: 2019-2020 aTox contributors
+// SPDX-FileCopyrightText: 2019-2025 Robin Lindén <dev@robinlinden.eu>
```

Comment:
Only you now? :P no more contributors.

Reply:
Yeah, it was always just me in this file, but I set all files to "aTox contributors" in the script that added them. :( I add back myself as I touch files.